dapp/src/components/Layout.tsx

```diff
@@ -1,27 +1,37 @@
 import React from "react"
 import { Outlet } from "react-router-dom"
 import { Flex } from "@chakra-ui/react"
-import { useMobileMode } from "#/hooks"
+import { useIsEmbed, useMobileMode } from "#/hooks"
 import DocsDrawer from "./DocsDrawer"
 import Header from "./Header"
 import ModalRoot from "./ModalRoot"
 import Sidebar from "./Sidebar"
+import MobileModeBanner from "./MobileModeBanner"
+
+const PAGE_MAX_WIDTH = {
+  standalone: "110rem", // 1760px
+  embed: "68rem", // 1088px
+}
 
 function Layout() {
   const isMobileMode = useMobileMode()
+  const { isEmbed } = useIsEmbed()
+
+  if (isMobileMode) return <MobileModeBanner forceOpen />
 
   return (
-    <Flex flexFlow="column">
+    <Flex
+      flexFlow="column"
+      p={10}
+      mx="auto"
+      maxW={isEmbed ? PAGE_MAX_WIDTH.embed : PAGE_MAX_WIDTH.standalone}
```

Comment:
We probably should set the `maxW` based on the embedded app. For example, the `maxW` may be different for Ledger and Xverse app. Non blocking comment.

Reply:
Yeah, you are right. I updated the code and added a `TODO` to update it when these changes are merged https://github.com/thesis/acre/pull/761/. Because this PR provides `embeddedApp` param to check the type of app.

Reply:
https://github.com/thesis/acre/pull/763/commits/8bfd363e67c19da09b4f6085116eddb7234fd662